src/fleet_app/error_result.cr

```diff
@@ -0,0 +1,15 @@
+module FleetApp
```

Comment:
do we deprecate this file now?

https://github.com/hostari/fleet_app_client/blob/main/src/fleet_app/error.cr

Reply:
Yes

Reply:
I mean, no.